guide/rotterdam/template_zhuoyu.qmd

```diff
@@ -0,0 +1,17 @@
+---
+title: "recommendation for visiting Rotterdam"
+description: "walk along the Nieuwe maas river"
+author:
+  - name: "joyce-shi-github"
+    url: "https://github.com/joyce-shi-github"
+# you can add more categories here (please add one category per line):
+categories:
+  - Template
```

Comment:
Add a meaningful category label (e.g., `Walk` or `Outdoor`) which replaces the `Template` category.